scripts/db_backup.sh

```diff
@@ -387,6 +396,52 @@ cleanup_old_remote_files()
     set -x
 }
 
+cleanup_old_TrackingLogs()
+{
+    if [ -z $BACKUP_RETENTIONDAYS ]
+    then
+        log "No database retention length provided."
+        return
+    fi
+
+    echo "Getting list of files and extracting their age"
+    echo "files older than $BACKUP_RETENTIONDAYS days will be removed"
+
+    # Calculate cutoff time.
+    logs_currentSeconds=$(date --date="`date`" +%s)
+    logs_retentionPeriod=$(( $BACKUP_RETENTIONDAYS * 24 * 60 * 60 ))
+    logs_cutoffInSeconds=$(( $logs_currentSeconds - $logs_retentionPeriod ))
+
+    # This is very noisy. We'll use log to communicate status.
+    set +x
+
+    # Get file list with lastModified meta-data. We'll use this to extract dates.
+    logs_verboseDetails= azure storage blob list $CONTAINER_NAME --json | jq -r '.[] | .name + "_" + .lastModified' > tracklogsblobs.json
+    
+    while read obj ; do
+        
+        # Split BlobName and BlobDate from below String Format; Split by underscore
+        # test1/test2/tracking.log-20170801-1501586221.gz_Sun, 07 Jan 2018 19:39:33 GMT
+        blobName=`echo $obj | cut -d _ -f -1`
+        blobDate=`echo $obj | cut -d _ -f 2-`
+        
+        logs_fileDateInSeconds=$(date --date="$blobDate" +%s)
+        #echo "logs_fileDateInSeconds - $logs_fileDateInSeconds"
```

Comment:
You may want to remove this.

Reply:
removed the comment
#echo "logs_fileDateInSeconds - $logs_fileDateInSeconds"